tests/test_scalar.py

```diff
@@ -33,7 +33,7 @@
     pytest.param("abc", "'abc'"),
     pytest.param(b"abc", "'abc'"),
     pytest.param("a'b''c'", "'a\\'b\\'\\'c\\''"),
-    pytest.param("a\\''b''c'", "'a\\'\\'b\\'\\'c\\''"),
+    pytest.param("a\\''b''c'", "'a\\\\\\'\\'b\\'\\'c\\''"),
```

Comment:
Maybe I am missing something obvious but why do we escape special characters with two backslashes instead of one when generating SnQL ?
Like can't we escape 
```
`
```
as 
```
\`
```
?

Reply:
That's what we do. In Python I have to escape the backslash again.